apps/emqx/src/emqx_misc.erl

```diff
@@ -557,16 +557,47 @@ try_to_existing_atom(Convert, Data, Encoding) ->
         _:Reason -> {error, Reason}
     end.
 
-is_sensitive_key(token) -> true;
-is_sensitive_key("token") -> true;
-is_sensitive_key(<<"token">>) -> true;
-is_sensitive_key(password) -> true;
-is_sensitive_key("password") -> true;
-is_sensitive_key(<<"password">>) -> true;
-is_sensitive_key(secret) -> true;
-is_sensitive_key("secret") -> true;
-is_sensitive_key(<<"secret">>) -> true;
-is_sensitive_key(_) -> false.
+is_sensitive_key(token) ->
+    true;
+is_sensitive_key("token") ->
+    true;
+is_sensitive_key(<<"token">>) ->
+    true;
+is_sensitive_key(password) ->
+    true;
+is_sensitive_key("password") ->
+    true;
+is_sensitive_key(<<"password">>) ->
+    true;
+is_sensitive_key(secret) ->
+    true;
+is_sensitive_key("secret") ->
+    true;
+is_sensitive_key(<<"secret">>) ->
+    true;
+is_sensitive_key([{str, Bin}]) when is_binary(Bin) ->
```

Comment:
Where does this come from? Maybe a short comment?

Reply:
A short comment is a good idea